src/Chainweb/Miner/Core.hs

```diff
@@ -268,7 +268,9 @@ callExternalMiner minerPath0 minerArgs saveStderr target blockBytes = do
       where
         act = do
             b <- B.hGet h 4000
-            if B.null b then return "stderr not saved" else act
+            if B.null b 
+              then return "See prior logs for error message" 
+              else B.hPutStr stderr b >> act
```

Comment:
We can just make this the definition of `slurp` instead of redirecting to `act`. Otherwise, 👍

Reply:
Personally I think we should just make `saveStderr` configurable?

Reply:
`saveStderr` is more for saving the error results, so that they can be read in tests IIRC.  You wouldn't want to enable it for miners. I could be wrong though

Reply:
I'd be happy to just make it configurable though.  Its a better solution for sure